packages/gatsby-theme-store/src/templates/product.browser.tsx

```diff
@@ -17,9 +18,7 @@ import type {
 export type BrowserProductPageProps = PageProps
 
 const ProductPage: FC<BrowserProductPageProps> = (props) => {
-  const {
-    params: { slug },
-  } = props
+  const { slug } = useMatch(props.pageResources.page.matchPath!)!
```

Comment:
why?

Reply:
the `find-path.js` changes how gatsby does routing. This removes the props.params feature of the gatsby framework and we need to manually match the path